jobs/mini-e2e.yaml

```
@@ -46,7 +54,7 @@
           status-url: $RUN_DISPLAY_URL
           status-context: 'ci/centos/mini-e2e/k8s-{k8s_version}'
           # yamllint disable-line rule:line-length
-          trigger-phrase: '/(re)?test ci/centos/mini-e2e(/k8s-{k8s_version})?'
+          trigger-phrase: '/(re)?test ci/centos/mini-e2e(/k8s-{k8s_version}-{test_type})?'
```

```suggestion
          trigger-phrase: '/(re)?test ci/centos/mini-e2e(/k8s-{k8s_version}(test_type-{test_type})?)?'
```
it should be optional and with prefix test-type ?

---

Yes, the intention was to keep it optional only. Updated now with the `/test_type-{test_type}` format.